apps/site/src/styles/craft-theme.css

```
@@ -0,0 +1,60 @@
+  --turbo-panel-border: #3d342a;
+  --turbo-dropdown-bg: #1a1612;
+  --turbo-dropdown-border: #3d342a;
+  --turbo-dropdown-item-hover: #221c17;
+  --gradient-primary: linear-gradient(135deg, #e8622a 0%, #c44e1a 100%);
+  --shadow-md: 0 4px 16px rgba(20, 16, 10, 0.35);
+  --shadow-lg: 0 8px 32px rgba(20, 16, 10, 0.45);
+  --shadow-glow: 0 0 0 3px color-mix(in srgb, #e8622a 30%, transparent);
+
+  --site-accent: #e8622a;
+  --site-accent-light: #d4b86a;
+  --font-display: "Fraunces", Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)